src/cosmicqc/analyze.py

```diff
@@ -174,15 +188,22 @@ def find_outliers(
     # Include metadata columns in the output DataFrame
     columns_to_include = list(feature_thresholds.keys()) + metadata_columns
 
+    result = outliers_df[columns_to_include]
+
+    # export the file if specified
+    if export_path is not None:
+        SCDataFrame(data=result).export(file_path=export_path)
+
```

Comment:
Do you need to make a new variable for result or could you just use the `outliers_df[columns_to_include]` directly in the function?

Reply:
Thanks good question - I created a new variable here to help with the multiple references. There's probably a way to reduce this, but it felt easier to understand in this format.